apps/assisted-ui/README.md

```diff
@@ -91,6 +91,14 @@ You can build the container image by running:
 $ podman build -t quay.io/edge-infrastructure/assisted-installer-ui:latest . --build-arg AIUI_APP_GIT_SHA="$(git rev-parse HEAD)" --build-arg AIUI_APP_VERSION=latest
 ```
 
+## ChatBot
+
+You can run the standalone UI with chatbot enabled. You need to be logged in via `ocm`.
+
+```
+$ AIUI_CHAT_URL=<chatbot_url> AIUI_OCM_TOKEN=$(ocm token) yarn start:assisted_ui
```

Comment:
```suggestion
$ AIUI_CHAT_API_URL=<chatbot_url> AIUI_OCM_TOKEN=$(ocm token) yarn start:assisted_ui
```
looks clearer to me and  more aligned with existing patterns for  `AIUI_APP_API_URL` etc